src/components/Testimonials/Testimonials.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import React from 'react'
+import withStyles from 'isomorphic-style-loader/lib/withStyles'
+import Slider from 'react-slick'
+
+import s from './Testimonials.css'
+
+type Props = {
+  testimonials?: Array<string>,
+}
+
+const Testimonials = ({ testimonials }: Props) => {
+  const settings = {
```

Comment:
you can put the settings outside the render, so we won't redeclare them every time the component is rendered

Reply:
@dbsolde have you seen this comment?